src/foundations/type/Type.constants.ts

```diff
@@ -0,0 +1,38 @@
+import {
+    TBodySize,
+    TTypeElement,
+    TTypeSize,
+    TTypeWeight,
+    TTypeMargin,
+    TTypeColor,
+} from './Type.types';
+
+const DEFAULT_TYPE_SIZE: TTypeSize = 100;
+
+const BODY_SIZES: TBodySize[] = [25, 50, 75, 100, 200, 300];
+
+const HEADING_SIZES: TTypeSize[] = [25, 50, 75, 100, 200, 300, 400, 500, 600, 700, 1000];
+
+const HEADING_ELEMENTS: TTypeElement[] = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'];
```

Comment:
I get why there are `HEADING_ELEMENTS` here and not `BODY_ELEMENTS`, it just seem weird to me at first.😛